src/app/component/followers/followers.component.html

```diff
@@ -0,0 +1,50 @@
+<div>
+  <h3 align="center">
+    🕸 My Followers
+  </h3>
+  <p align="center">You can see all the followers here👇</p>
+</div>
+
+<div class="clr-row clr-justify-content-center">
+<div *ngIf="(loadingFollowers)" class="clr-row clr-justify-content-center">
+  <div class="alert-items">
+      <div class="alert-item static clr-justify-content-center">
+          <span class="alert-text">Loading followers, please wait...</span>
+      </div>
+  </div>
+</div>
+
+  <div *ngIf="!(loadingFollowers) && userProfiles?.length === 0"  class="clr-row clr-justify-content-center">
+    <div class="alert-items">
+        <div class="alert-item static clr-justify-content-center">
+          <span class="alert-text">No followers yet. May be try following some people? <br/> Like <a href="https://snort.social/p/npub1ltx67888tz7lqnxlrg06x234vjnq349tcfyp52r0lstclp548mcqnuz40t">@vivgan</a>?</span>
+        </div>
+    </div>
+  </div>
+</div>
+
+<div *ngFor="let userProfile of userProfiles">
+  <div class="card">
+    <div class="card-block">
+      <div class="card-media-block">
+        <img *ngIf="userProfile?.image" (click)="openProfileInSnort(userProfile)" alt="Profile picture" src="{{ userProfile?.image }}" class="card-media-image pointer-cursor" title="Open in Snort"/>
+        <img alt="temp avatar"
+          (click)="openProfileInSnort(userProfile)"
+          *ngIf="userProfile?.image == undefined"
+          class="card-media-image pointer-cursor"
+          src="https://robohash.org/{{ event?.pubkey }}.png?size=100x100"
+          title="Open in Snort" />
+          <div class="card-media-description">
+            <span class="card-media-title break-words-overflow pointer-cursor "  title="Open in Snort" (click)="openProfileInSnort(userProfile)"> <strong>{{ userProfile?.displayName }} </strong></span>
+            <span *ngIf="!userProfile?.displayName" class="card-media-title break-words-overflow pointer-cursor"  title="Open in Snort" (click)="openProfileInSnort(userProfile)"> {{ event?.pubkey }} </span>
```

Comment:
we can remove the open in snort thing as we havn't implemented yet.

Any PR approved will immediately get pushed to the prod env. so we dont have a way to accept WIP